src/lib/validations/sql.ts

```diff
@@ -337,7 +337,7 @@ const checks: Checks = {
 		recommendation: "Use search-replace to change environment's domain",
 	},
 	engineInnoDB: {
-		matcher: / ENGINE=(?!(InnoDB))/i,
+		matcher: / ENGINE\s?=(?!(\s?InnoDB))/i,
```

Comment:
Wonder why there's a mandatory whitespace in the beginning?

Reply:
I don't think so. If there's no space in the beginning then `ENGINE` would turn into a totally different word wouldn't it?

Reply:
oh I thought it always expects it on a newline.

What if the `ENGINE=MyISAM` clause is on a completely newline?

Reply:
Hmm.. that's a good point. I'll test it out

Reply:
Changed in a1c7999379dc4eb8c4b24d6e7a25bd8138c56945
I've added a requirement for there to be at least 1 whitespace character before `ENGINE`. This includes space, tab and newline characters

Reply:
@hanifn can't see the commit against this PR for some reason, but it looks good